packages/grid/_modules_/grid/hooks/features/filter/useGridFilter.ts

```diff
@@ -404,6 +335,37 @@ export const useGridFilter = (
     'FilterApi',
   );
 
+  /**
```

Comment:
I started to structure the big hooks with section comments to make it easier to find something.